src/silx/gui/hdf5/Hdf5TreeModel.py

```diff
@@ -250,7 +250,6 @@ def _closeFileList(fileList):
         """Static method to close explicit references to internal objects."""
         _logger.debug("Clear Hdf5TreeModel")
         for obj in fileList:
-            _logger.debug("Close file %s", obj.filename)
```

Comment:
This was causing issues with the tests since `h5py.Dataset` do not have a `filename` attribute.